web/src/components/netflow-traffic-parent.tsx

```diff
@@ -52,9 +53,11 @@ class NetflowTrafficParent extends React.Component<Props, State> {
     // else render default NetworkTraffic
 
     return (
-      <AlertFetcher>
-        <NetflowTraffic forcedFilters={null} />
-      </AlertFetcher>
+      <DynamicLoader>
+        <AlertFetcher>
+          <NetflowTraffic forcedFilters={null} />
+        </AlertFetcher>
+      </DynamicLoader>
```

Comment:
`DynamicLoader` FC is loaded on top of everything to ensure the hooks are up to date.